WordPress/Classes/Models/Post.swift

```diff
@@ -195,7 +195,7 @@ class Post: AbstractPost {
 
         if let originalPost = original as? Post {
 
-            if tags?.characters.count != originalPost.tags?.characters.count && tags != originalPost.tags {
+            if tags ?? "" != originalPost.tags ?? "" {
```

Comment:
While the logic used here is fine, readability could be improved.

Here's one example using the exact same code, but making the intent of developer clear and explicit (for any future readers):

```swift
let tagsHaveLocalChanges = tags ?? "" != originalPost.tags ?? ""

if tagsHaveLocalChanges {
```

A step further could be to abstract the details:

```swift
if tagsHaveLocalChanges() {
```

The solutions above are a step closer to something like this (which in my opinion would do this method a big favor):

```swift
override func hasLocalChanges() -> Bool {
    if super.hasLocalChanges() {
        return true
    }
     
    assert(original == nil || original is Post)

    if let originalPost = original as? Post,
        tagsHaveLocalChanges() || coordsHaveLocalChanges() {
           return true
    }

    return false
}
```